src/functions/def.js

```diff
@@ -65,12 +114,66 @@ defineFunction({
             }
             arg = parser.gullet.consumeArgs(1)[0];
         }
+        if (funcName === "\\edef" || funcName === "\\xdef") {
+            arg = parser.gullet.expandTokens(arg);
+            arg.reverse();
+        }
         // Final arg is the expansion of the macro
         parser.gullet.macros.set(name, {
             tokens: arg,
             numArgs,
-        }, funcName === "\\gdef");
+        }, funcName === globalMap[funcName]);
+
+        return {
+            type: "internal",
+            mode: parser.mode,
+        };
+    },
+});
 
+// <simple assignment> -> <let assignment>
+// <let assignment> -> \futurelet<control sequence><token><token>
+//     | \let<control sequence><equals><one optional space><token>
+// <equals> -> <optional spaces>|<optional spaces>=
```

Comment:
Is this a description of possible expansions?  ~Where do we match the `equals`?~ I see, `getRHS` handles this.